release_notes/ocp-4-9-release-notes.adoc

```diff
@@ -408,6 +408,22 @@ Kubernetes 1.22 introduces link:https://kubernetes.io/docs/reference/using-api/d
 When you uninstall an Operator by using the web console, you can now remove Operands managed by the Operator automatically, including custom resources (CRs). For more information, see
 xref:../operators/admin/olm-deleting-operators-from-cluster.adoc#olm-deleting-operators-from-a-cluster-using-web-console_olm-deleting-operators-from-a-cluster[Deleting Operators from a cluster using the web console].
 
+[id="ocp-4-9-olm-admin-error-reporting"]
+==== Enhanced error reporting for cluster administrators
+Because administrators should not require understanding of the interaction process between the various low-level APIs or access to the OLM pod logs to successfully debug such issues, {product-title} {product-version} introduces the following enhancements in Operator Lifecycle Manager (OLM) to provide administrators with more comprehensible error reporting and messages:
+
+[id="ocp-4-9-olm-operator-group-status-conditions"]
+Updating Operator group status conditions::
+Previously, if a namespace contained multiple Operator groups or could not find a service account, the status of the Operator group would not report an error. With this enhancement, these scenarios now update the status condition of the Operator group to report an error.
```

Comment:
This text is rendering weird on the portal. You can see that the body text is indented. I'm not sure why that is, but it may have to do with the missing ==== at the heading? I also just noticed it's done like that in 4.8, so I fixed it real quick. It looks like adding ==== in the heading and removing the two colons at the end fixes the issue.